pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,32 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
+
+	if follow {
```

Comment:
@hferentschik ,  Well out of curiosity during weekend I was just going through 'logs' command and found `docker logs origin -f` is the underline command we need. So, then the main crux was how to streaming it as you mention. 

Then, I looked into [minikube code](https://github.com/kubernetes/minikube/blob/master/pkg/minikube/cluster/cluster.go#L383-L393) which seems to be pretty easy to modify and adapt here too.